forge/routes/api/shared/users.js

```diff
@@ -35,32 +35,37 @@ module.exports = {
                 }
 
                 if (request.body.suspended !== undefined) {
-                    if (request.body.suspended === true) {
-                        await app.db.controllers.User.suspend(user)
-                        if (app.postoffice.enabled()) {
-                            // Send email
-                            const context = {}
-                            if (app.config.support_contact) {
-                                context.support = app.config.support_contact
-                            } else {
-                                const admin = await app.db.models.User.scope('admins').findOne()
-                                if (admin?.email) {
-                                    context.support = `mailto:${admin.email}`
+                    if (request.user.id !== user.id) {
```

Comment:
Missed this in the review - `request.user` is the user being updated - which at the same point is the same as `user`.

This should be `request.session.User.id !== user.id`. In hindsight I now see the unit test was verifying admin cannot suspend themselves, but we didn't have a test verifying admin can suspend *anyone else*. Ooops.

Fix incoming as part of #463